README.md
# Open-Data-MeadoWatch Project

This is the repository for the analyses and data of the data paper of the MeadoWatch project: LINK TO THE DATA PAPER TO BE ADDED HERE

THIS REPOSITORY WILL REMAIN UNCHANGED FOR CONSISTENCY WITH THE ANALYSES IN THE ABOVEMENTIONED LINK. AN UP-TO-DATE VERSION OF THE MEADOWATCH DATABASE IS AVAILABLE IN THE OFFICIAL REPOSITORY HERE: LINK TO BE ADDED.

MeadoWatch director and contact: Prof. Dr. Janneke Hille Ris Lambers. Institute of Integrative Biology, D-USYS, ETH Zürich
MeadoWatch coordinators:
Dr. Meera Sethi. Biology Department, University of Washington, Seattle
Prof. Dr. Elli Theobald. Biology Department, University of Washington, Seattle
Manuscript analyses and lead:
Dr. Rubén D. Manzanedo. Institute of Integrative Biology, D-USYS, ETH Zürich
PhD Candidate Aji John. Biology Department, University of Washington, Seattle
Berry Brosi. Biology Department, University of Washington, Seattle
Joshua Jenkins. Biology Department, University of Washington, Seattle
Emilia Lia. Biology Department, University of Washington, Seattle
Annie Schiffer. Biology Department, University of Washington, Seattle
Jordana Sevigny. Biology Department, University of Washington, Seattle
Anna Wilson. Biology Department, University of Washington, Seattle
Yonit Yogev. National Park Service. USA.


Permanent website of MeadoWatch and further info: http://www.meadowatch.org/
https://www.youtube.com/channel/UCGBFTKxf8FIWswMDxBavpuQ/featured


PUBLICATIONS LINKED TO THE MEADOWATCH DATABASE
- Hille Ris Lambers, J., Cannistra, A. F., John, A., Lia, E., Manzanedo, R. D., Sethi, M., Sevigny, J., Theobald, E. J., Waugh, J. K. (2021) Climate change impacts on natural icons: do phenological shifts threaten the relationship between peak wildflowers and visitor satisfaction? Climate Change Ecology, 100008.

- Breckheimer, I. K., Theobald, E. J., Cristea, N. C., Wilson, A. K., Lundquist, J. D., Rochefort, R. M., & HilleRisLambers, J. (2020). Crowd‐sourced data reveal social–ecological mismatches in phenology driven by climate. Frontiers in Ecology and the Environment, 18(2), 76-82.

- John, A., Ong, J., Theobald, E. J., Olden, J. D., Tan, A., & HilleRisLambers, J. (2020). Detecting Montane Flowering Phenology with CubeSat Imagery. Remote Sensing, 12(18), 2894.

- Wilson, A., Bacher, K., Breckheimer, I., Lundquist, J., Rochefort, R., Theobald, E., ... & HilleRisLambers, J. (2017). Monitoring wildflower phenology using traditional science, citizen science, and crowd sourcing. Park Sci, 33, 17-26.

# Repository content
RAW DATA:
- MW_PhenoDat_2013_2019.csv -- Phenological records. Information on date, transect id, observer, and phenological state per flowering species.

- MW_SiteInfo_2013_2019.csv -- Latitude and longitude locatoin (WGS84), elevation and forest type for each MW site

- MW_Phenocurves.csv -- 

- MW_ground_temperatures.csv --


R SCRIPTS:
- Part 1 - Location temporal replication and species comp.R - Analyses and plotting for figure 1. 
- Part 2 - species composition.R - Analyses for Figure 2b.
- Part 3 - temporal resolution.R - Analyses for Figure 3.
- Part 4 - comparison scientists citizens.R - Functions and selecting for Sci-cit comparison.
- Part 5 - figure accuracies sci-cit.R - Analyses and plotting for figure 5.

FIGURES (SVG and PNG):
- All in folder 'Exported figures'. Exported figures from R Studio has been finalized in Inkscape. Basic versions of all figures can be done with the accompanying code,

MANUSCRIPT
- First draft_data_paper_MW.docx - Manuscript versions (currently in google docs: https://docs.google.com/document/d/101CBT2GUFirWMFm_pHwYSdolv_xV4OsDa3H95RNpcmU/edit?usp=sharing)


www/releases.md
### Releases
<br>
<p style=" font-size:14px; text-align:justify;">
</br>
<b>version:</b> 1.0
<ul style="padding-left:5em">
 <li type="square">initial release /1st of January 2021&nbsp;&nbsp;</li>
</ul></p2>
<b>version:</b> 1.1
<ul style="padding-left:5em">
 <li type="square">Data paper release /1st of July 2021&nbsp;&nbsp;</li>
</ul></p2>
</p>



Part 2 - species composition.R
############################DATA PAPER MEADOWATCH###################
###DATA COMPILED BY AJI JOHN
###FIGURES AND THIS ANALYSIS: RUBÉN D. MANZANEDO. APRIL-JULY 2021#########

###################PART 2######################

##Figure 2: Species composition of the plots
#load report data
MW_PhenoDat_2013_2019 <- read.csv("MW_PhenoDat_2013_2019.csv")
head(MW_PhenoDat_2013_2019)

#graphical parameters
par(bty="o")
par(lwd=2)
par(tcl=-0.2)
par(las=1)
par(cex.lab=1.2)


#keep the reports
species.dist = data.frame('species' = MW_PhenoDat_2013_2019$Species,
                          'observer' = MW_PhenoDat_2013_2019$Observer,
                          'transect' = MW_PhenoDat_2013_2019$Transect)

#remove all other duplicates
unique.species.dist= unique(species.dist)

#calculate the proportion of species per trail
rep.species = table(unique.species.dist[,-2])
rep.species2 = prop.table(rep.species,2)

#plot as a vector plot
barplot(rep.species2,
        col = rainbow(dim(rep.species2)[1]),
        ylab='Proportion of reports per species (%)')

#define the x position and the color coding to add labels on top
acc.pos.gb = cumsum(rep.species2[,1])
acc.pos.rl = cumsum(rep.species2[,2])
acc.lab.gb = rep.species2[,1] != 0
acc.lab.rl = rep.species2[,2] != 0

#labels appear on top (will me adjusted in inkscape, zero presence are wriTten white)
text(rownames(rep.species2), 
     x=rep(1, length(rownames(rep.species2))),
     y=acc.pos.gb,
     col=acc.lab.gb)
text(rownames(rep.species2), 
     x=rep(2, length(rownames(rep.species2))),
     y=acc.pos.rl,
     col=acc.lab.rl)

#calculate species per plot and then replace the elevations

MW_PhenoDat_2013_2019$Site_Code = as.factor(MW_PhenoDat_2013_2019$Site_Code)
levels(MW_PhenoDat_2013_2019$Site_Code) = c("GB01","GB10","GB11","GB12","GB13","GB14",
                                            "GB15","GB16","GB17","GB02","GB03","GB04",
                                            "GB05","GB06","GB07","GB08","GB09","RL01","RL10","RL11",
                                            "RL02","RL03","RL03a","RL03a","RL03b","RL04","RL05","RL05a",
                                            "RL05b","RL05b","RL06","RL07","RL08","RL09")

species.dist2 = data.frame('species' = MW_PhenoDat_2013_2019$Species,
                          'observer' = MW_PhenoDat_2013_2019$Observer,
                          'transect' = MW_PhenoDat_2013_2019$Transect,
                          'site_Code' = MW_PhenoDat_2013_2019$Site_Code)

#remove duplicates
unique.species.dist2= unique(species.dist2)

#calculate the proportion of species per plot and by proportion
alt.species = table(unique.species.dist2[,-c(2,3)])

#plots RL08 and RL09 have no records?
alt.species = alt.species[,-c(31,32)]
alt.species.prop = prop.table(alt.species,2)

#divide RL and GB
GB.species = alt.species[,1:17]
RL.species = alt.species[,18:30]

#sort them in increasing value fo the plots
GB.ordered = GB.species[,order(colnames(GB.species))]
RL.ordered = RL.species[,order(colnames(RL.species))]

#prepare color palettes and plot 
coloring = c('white','white','orange', 'darkorange')
heatmap(GB.ordered,col = coloring, Colv = NA, Rowv=NA, scale='column')

coloring = c('white','dodgerblue3', 'dodgerblue4')
heatmap(RL.ordered, col=coloring, Colv = NA, Rowv=NA, scale='column')

#change order and remove not present species?
totals.GB= apply(MARGIN=1,GB.ordered, FUN='sum')
totals.RL= apply(MARGIN=1,RL.ordered, FUN='sum')

#####END OF CODE#########

